source/packaging-guide/manual.rst

```diff
@@ -27,7 +27,7 @@ Create an AppDir structure that looks (as a minimum) like this::
 	MyApp.AppDir/usr/bin/myapp
 	MyApp.AppDir/usr/lib/libfoo.so.0
 
-The :code:`AppRun` file can be a script or executable. It sets up required environment variables such as :code:`$PATH` and launches the payload application. You can write your own, but in most cases it is easiest (and most error-proof) to use a precompiled one from this repository.
+The :code:`AppRun` file can be a script or executable. It sets up required environment variables such as :code:`$PATH` and launches the payload application. You can write your own, but in most cases it is easiest (and most error-proof) to use a precompiled one from `this repository <https://github.com/AppImage/AppImageBuild>`__.
```

Comment:
That would have been AppImageKit (or, rather, its release page) if anything. However, this section is deprecated. AppImageKit's AppRun often does more harm than good. There should be some words of warning. Something like "... this one might work, but please note its limitations: [list]`.

We should open an issue about the latter point.

Reply:
Hmmm... well I have no idea what the limitations are.

Reply:
@TheAssassin I ran across this issue today as well. Followed the documentation, which advised me to use the AppRun binary, which I then needed to search for.

But if that's not actually what should be used, is there some recommended launch script somewhere?

Reply:
This is not the right place for end user support. Feel free to join our chat: https://docs.appimage.org/contact.html